ajax/libs/Bounce/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "bounce",
```

Comment:
The value of name key needs to as the same with the folder name.
Also, let's use `bounce.js` as the name and the folder name is `ajax/libs/bounce.js/package.json`.
Because making this lib's name as the upstream can help people find this lib quickly.
Many thanks.